code/src/java/pcgen/cdom/enumeration/AssociationKey.java

```diff
@@ -126,8 +126,7 @@ public T cast(Object obj)
 		return (T) obj;
 	}
 
-	public static <OT> AssociationKey<OT> getKeyFor(Class<OT> assocClass,
-			String assocName)
+	public static <OT> AssociationKey<OT> getKeyFor(String assocName)
```

Comment:
Interesting. I was unaware of this quirk, just did some reading about protecting against erasure here. Will revert.